app/controllers/infos_controller.rb

```diff
@@ -0,0 +1,27 @@
+class InfosController < ApplicationController
+
+	def index
+		@infos = Info.all
```

Comment:
try and change your settings from 4 spaces to two spaces for ruby

Reply:
I don't think we need an index for all of the infos since we will be looking at all the tenants for a RA not their infos